ckan_sdk/f11s.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
What happened to reusing the datapackage-py library?

Reply:
Not absolutely required but i think you will save yourself some energy longer term ...  (maybe).

Reply:
The Resource Class in the `datapackage` accepts a path of a JSON file. in the constructor. And it is also not providing the data for a resource file that we are requiring from 'load()` currently method.

Don't know much but I found out confusing. Than's why made my own. 
But if you have better knowledge about that datapackage, can you guide me, please?
As you are saying that it will save me some energy for the longer term.